### ISSUE TITLE
Schedulers ignores delay parameters? 

### ISSUE THREAD

**Issue by fromgate** [Nukkit/Nukkit#158]
I created a command for tests:
```
	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		Server.getInstance().getLogger().info("Current time: "+System.currentTimeMillis());
		this.getServer().getScheduler().scheduleDelayedTask(new Runnable(){
			@Override
			public void run() {
				Server.getInstance().getLogger().info("Time after 200 ticks: "+System.currentTimeMillis());
			}
		}, 200); // 200 ticks = 10 seconds???
		this.getServer().getScheduler().scheduleDelayedTask(new Runnable(){

			@Override
			public void run() {
				Server.getInstance().getLogger().info("Time after 1 tick: "+System.currentTimeMillis());
			}
		}, 1); // 1 tick
		Server.getInstance().getLogger().info("Current time 2: "+System.currentTimeMillis());
		return true;
	}

```

And here is output at server console:
```
> test
23:52:20 [INFO] Current time: 1450039940329
23:52:20 [INFO] Current time 2: 1450039940331
23:52:20 [INFO] Time after 200 ticks: 1450039940336
23:52:20 [INFO] Time after 1 tick: 1450039940337
```

Looks like there is no differences between 1 tick delay and 200 ticks delay...

**Comment by caoli5288** [comment]
Fixed #159 .Wait merge :-)

**Comment by fromgate** [comment]
Thank you!
